src/app/common/components/modals/dialog-create-or-update-wallet/wallet-form-modal.component.ts

```diff
@@ -0,0 +1,25 @@
+import { Component, Inject } from '@angular/core';
+import { MatDialogRef, MAT_DIALOG_DATA }  from '@angular/material/dialog';
+import { IWalletModalData } from 'src/app/domains/wallets/domains.wallets.types';
+
+@Component({
+  selector: 'app-wallet-form-modal',
+  templateUrl: './wallet-form-modal.component.html',
+  styleUrls: ['./wallet-form-modal.component.scss']
+})
+export class WalletFormModalComponent {
+
```

Comment:
This empty line can be removed. Please check other files also if they need this adjustment.